README.rst

```diff
@@ -79,6 +79,14 @@ at creation time, so that this value will be used by default.
 
 .. code-block:: python
 
+    from fxa.plugins.requests import FxABearerTokenAuth
+    auth = FxABearerTokenAuth(
+        email, passwd,
+        scopes=['kinto'],
+        client_id="<FXA-CLIENT-ID>",
+        account_server_url='https://api.accounts.firefox.com/v1',
+        oauth_server_url='https://oauth.accounts.firefox.com/v1',
+    )
     client = Client(bucket="payments", collection="receipts", auth=auth)
```

Comment:
Shouldn't this be a separate example, rather than trying to wedge this in with the example where you can pass `bucket` and `collection` to the `Client` constructor?